libs/shared/console-shared/src/lib/environment-buttons-actions/ui/environment-buttons-actions.tsx

```diff
@@ -167,23 +168,23 @@ export function EnvironmentButtonsActions(props: EnvironmentButtonsActionsProps)
         topItems.push(stopButton)
       }
 
-      // todo: not implemented yet on v3
-      //   const updateAllButton = {
-      //     name: 'Update all services',
-      //     contentLeft: <Icon name={IconAwesomeEnum.ROTATE} className="text-sm text-brand-400" />,
-      //     onClick: (e: ClickEvent) => {
-      //       e.syntheticEvent.preventDefault()
-      //
-      //       openModalConfirmation({
-      //         mode: environment.mode,
-      //         title: 'Confirm update all',
-      //         description: 'To confirm the update of all the services of your environment, please type the name:',
-      //         name: environment.name,
-      //         action: () => console.log('todo: update all'),
-      //       })
-      //     },
-      //   }
-      // bottomItems.push(updateAllButton)
+      const updateAllButton = {
+        name: 'Update all services',
+        contentLeft: <Icon name={IconAwesomeEnum.ROTATE} className="text-sm text-brand-400" />,
+        onClick: (e: ClickEvent) => {
+          e.syntheticEvent.preventDefault()
+          //projectId,
+          //environmentId: environment.id,
```

Comment:
You can remove this one

BTW on this file, you have a warning here 
<img width="946" alt="Capture d’écran 2023-01-17 à 15 16 59" src="https://user-images.githubusercontent.com/533928/212922102-2ee9ac98-033e-43ad-b875-357e94b0c21c.png">